src/v3/src/hooks/usePolling.ts

```diff
@@ -100,6 +107,19 @@ export const usePolling = (
         stateHandle: stateToken && idxTransaction?.context?.stateHandle,
         ...payload,
       });
+
+      // @ts-expect-error OKTA-585869 errorCode & errorIntent properties missing from context type
+      if (newTransaction?.context?.errorCode === 'E0000047' && !newTransaction?.context?.errorIntent) {
```

Comment:
catch this unhandled error before it sets to the transaction so we can extend the polling refresh time and continue polling